src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -67,6 +69,21 @@ class WorkspaceMembersPage extends React.Component {
         this.hideConfirmModal = this.hideConfirmModal.bind(this);
     }
 
+    componentDidMount() {
+        const clientPolicyMembers = _.keys(this.props.policyMemberList);
+        Policy.openWorkspaceMembersPage(this.props.policy.id, clientPolicyMembers);
```

Comment:
Let's use lodashGet any time we're grabbing stuff from the policy, i.e.

```
lodashGet(this.props.policy, 'id', '');
```

Reply:
Using `this.props.route.params.policyID` instead since that is always safe